src/pages/ChattingRoom.tsx

```diff
@@ -0,0 +1,209 @@
+import { useState, useEffect, useRef } from "react";
+import { useRecoilState, useRecoilValue } from "recoil";
+import {
+  currentUsersState,
+  typingUserState,
+  // currentChatListState,
+} from "../recoil/recoil";
+import styled from "styled-components";
+import SmallButton from "../components/SmallButton";
+import Profile from "../components/Profile";
+import LeftChat from "../components/LeftChat";
+import RightChat from "../components/RightChat";
+import { chatInterface } from "../interfaces/interface";
+import chatsData from "../json/chatsData.json";
+import Alert from "../components/Alert";
+
+const ChattingRoom = () => {
+  const currentUsers = useRecoilValue(currentUsersState);
+  const [typingUser, setTypingUser] = useRecoilState(typingUserState);
+  const nonTypingUser = currentUsers.filter(
+    (user) => user.userId !== typingUser.userId
+  )[0];
+  const [chatList, setChatList] = useState<chatInterface[]>(
+    chatsData.chattings[0].chatList
+  ); // TODO - recoil로 변경
+  // const [chatList, setChatList] = useRecoilState(currentChatListState);
+
+  const [inputText, setInputText] = useState<string>("");
+  const isTyping: boolean = inputText.trim() ? true : false;
+
+  const [isVisibleAlert, setIsVisibleAlert] = useState<boolean>(false);
+
+  const textareaRef = useRef<HTMLTextAreaElement>(null);
+  const bottomDivRef = useRef<HTMLDivElement>(null);
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setInputText(e.target.value);
+  };
+
+  const handleEnter = (e: React.KeyboardEvent<HTMLTextAreaElement>) => {
+    if (!e.shiftKey && e.key === "Enter") {
+      handleSubmit();
+    }
+  };
+
+  const handleSubmit = (e?: React.FormEvent<HTMLFormElement>) => {
+    e?.preventDefault();
+    setChatList([
+      ...chatList,
+      {
+        userId: typingUser.userId,
+        message: inputText,
+        date: String(new Date()),
+        chatId: `ex${chatList.length}`,
+      },
+    ]);
+    setInputText("");
+    textareaRef.current?.focus();
+  };
+
```

Comment:
```suggestion
  const handleSubmit = (e?: React.FormEvent<HTMLFormElement>) => {
    e?.preventDefault();
    if (inputText.trim()){
    setChatList([
      ...chatList,
      {
        userId: typingUser.userId,
        message: inputText,
        date: String(new Date()),
        chatId: `ex${chatList.length}`,
      },
    ]);
    setInputText("");
    textareaRef.current?.focus();
  }
  else{
    setInputText('');
    alert('chek input plz');
  }
  };
```

이 부분에서 `inputText`를 체크하면 공백 처리 할 수 있을 거 같아요!

Reply:
아 제가 예외처리를 놓쳤네요,,,, 이럴수가 ㅜㅜ
리뷰 주신대로 고쳐봤습니다! 역시 프운 👍👍